swap/simulations_test.go

```diff
@@ -198,7 +198,7 @@ func newSharedBackendSwaps(t *testing.T, nodeCount int) (*swapSimulationParams,
 		}
 		keys[i] = key
 		addrs[i] = crypto.PubkeyToAddress(key.PublicKey)
-		alloc[addrs[i]] = core.GenesisAccount{Balance: big.NewInt(10000 * int64(RetrieveRequestPrice))}
+		alloc[addrs[i]] = core.GenesisAccount{Balance: big.NewInt(9000000000000000000)}
```

Comment:
not sure i understand this change, could you please elaborate on this one? 

why `9000000000000000000`? does it make sense to extract this to a constant since it appears further down as well?

Reply:
It makes no sense for the premine to be a multiple of the `RetrieveRequestPrice` since we don't pay in ether anymore. `9000000000000000000` was chosen because it's a round number close to the max of `int64` that provides enough ether for all the gas usage.

I extracted it into a variable.